pyexcelerate/Worksheet.py

```diff
@@ -79,7 +80,15 @@ def num_rows(self):
 	@property
 	def num_columns(self):
 		return max(1, self._columns)
-	
+
+	@property
+	def show_gridlines(self):
```

Comment:
Can you call this `show_grid_lines` instead of `show_gridlines`? I know it's a nitpicky point, but it matches the xml attribute better. Thanks!